### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by m-mohr**
Processes, the final release. We can still add processes later, but existing processes can not be changed in a backward incompatible way. So better check it now. :-)

To be merged and released on Friday (31.07) morning.

### CODE REVIEW

File: aggregate_temporal.json [hunk 1]
```diff
@@ -150,80 +157,61 @@
...
-                                    "process_id": "mean",
-                                    "arguments": {
-                                        "data": {
-                                            "from_parameter": "data"
-                                        }
-                                    },
-                                    "result": true
+                "intervals": [
+                    [
+                        "2015-01-01",
```
Comment (neteler) [inline aggregate_temporal.json:174]: Minor comment (feel free to ignore):

Maybe a bit late, but perhaps shift the date range by one year towards 2020 for a better potential data coverage with Sentinel-2?

File: aggregate_temporal_period.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            "name": "dimension",
+            "description": "The name of the temporal dimension for aggregation. All data along the dimension is passed through the specified reducer. If the dimension is not set or set to `null`, the data cube is expected to only have one temporal dimension. Fails with a `TooManyDimensions` error if it has more dimensions. Fails with a `DimensionNotAvailable` error if the specified dimension does not exist.",
+            "schema": {
+                "type": [
+                    "string",
+                    "null"
+                ]
+            },
+            "default": null,
+            "optional": true
```
Comment (neteler) [inline aggregate_temporal_period.json:78]:
```suggestion
            "optional": true,
            "default": null
```

(same order also here)

File: aggregate_temporal_period.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        "DimensionNotAvailable": {
+            "message": "A dimension with the specified name does not exist."
+        },
+        "DistinctDimensionLabelsRequired": {
+            "message": "The dimension labels have duplicate values. Distinct labels must be specified."
+        }
+    },
+    "links": [
+        {
+            "href": "https://open-eo.github.io/openeo-api/glossary/#aggregation-and-resampling",
```
Comment (neteler) [inline aggregate_temporal_period.json:110]:
```suggestion
            "href": "https://openeo.org/documentation/1.0/glossary.html#aggregate-reducing-resolution",
```

https://open-eo.github.io/openeo-api/glossary/ -> 404

File: aggregate_temporal.json (hunk 1 above)
Comment (m-mohr) [reply aggregate_temporal.json:174]: The example is not specific to any data set (any longer), but if you think that would give a user a benefit, I'm happy to add a year.

File: aggregate_temporal_period.json (hunk 3 above)
Comment (m-mohr) [reply aggregate_temporal_period.json:110]: Thanks for pointing out, I'll fix that.

File: aggregate_temporal_period.json (hunk 2 above)
Comment (m-mohr) [reply aggregate_temporal_period.json:78]: Order in JSON is not relavant and I doubt we have a consistent order in the processes.

File: aggregate_temporal_period.json (hunk 3 above)
Comment (m-mohr) [reply aggregate_temporal_period.json:110]: I just realized there are multiple (6) similar links that are broken. Will fix them all.

File: aggregate_temporal_period.json (hunk 2 above)
Comment (m-mohr) [reply aggregate_temporal_period.json:78]: Changed it anyway

File: aggregate_temporal.json (hunk 1 above)
Comment (m-mohr) [reply aggregate_temporal.json:174]: Added

Comment (neteler) [reply aggregate_temporal.json:174]: Whatever makes sense - my motivation was that user may try it with Sentinel-2. No strong opinion here!

File: aggregate_temporal_period.json (hunk 3 above)
Comment (m-mohr) [reply aggregate_temporal_period.json]: Fixed them, thanks for the review.

File: .travis.yml [hunk 4]
```diff
@@ -8,34 +8,34 @@ sudo: false
...
+    mkdir -p gh-pages/draft
+    rsync -vrm --include='*.json' --include='*.html' --include='examples/***' --include='meta/***' --exclude='*' . gh-pages/draft
   fi
 - |
-  if [[ $TRAVIS_BRANCH == 'draft' ]] ; then
-    rm -rf draft
-    mkdir -p draft
-    cp ../index.html draft/index.html
-    cp ../processes.json draft/processes.json
+  if [[ $TRAVIS_BRANCH == 'master' ]] ; then
```
Comment (m-mohr) [inline .travis.yml:28]:
```suggestion
  if [[ $TRAVIS_BRANCH == 'wer' ]] ; then
```

**Comment by m-mohr** [comment]
PR is outdated, will directly merge.

**Pull request closed without merging**